flag_engine/organisations/models.py

```diff
@@ -8,3 +8,6 @@ class OrganisationModel:
     feature_analytics: bool
     stop_serving_flags: bool
     persist_trait_data: bool
+
+    def get_unique_slug(self):
```

Comment:
Couldn't this be a property? Or do we have to maintain the interface in the django app?

Reply:
If it's a property, let's name it accordingly, `unique_slug` is fine, you don't need the `get`.